src/index.js

```diff
@@ -1,6 +1,9 @@
 import { months } from './data.js'
+
 (async function () {
-  let setTimerId
+  let setTimerId, setTimerIdBottom
+  let bottomItemCount = 0
+  const bottomContainerElements = document.getElementsByClassName('bottom-itemscontainer')[0]
```

Comment:
Let's prefer using `querySelector` instead of `getElementsByClassName`.
`querySelector` will select the very first occurance of the selector and as we're only using one class named `'bottom-itemscontainer'` we need not get 0th item after selecting.